inspector/plottools/doDockerPyplot

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# doDockerPyplot [build] [shell]
+
+set -e
+DOCKER_TAG=measurementlab/nbrunner
+
+# deduce the user credentials: fail if not exactly one
+shopt -s nullglob
```

Comment:
Please set shell options before you do anything else, including create variables.

Reply:
Done